src/test/scala/com/fulcrumgenomics/umi/ExtractUmisFromBamTest.scala

```diff
@@ -159,6 +184,24 @@ class ExtractUmisFromBamTest extends UnitSpec with OptionValues {
     }
   }
 
+  it should "accept duplicate molecular barcode SAM tags and concatenate the values for fragment reads" in {
+    val output  = newBam
+    val builder = new SamRecordSetBuilder(readLength=100)
+    builder.addFrag(name="Frag1", start=1, unmapped=true).map { rec => rec.setReadString("A" * 100); rec }
+    builder.addFrag(name="Frag2", start=1, unmapped=true).map { rec => rec.setReadString("C" * 100); rec }
+
+    new ExtractUmisFromBam(input=builder.toTempFile(), output=output, readStructure=Seq("10M80T10M"), molecularBarcodeTags=Seq("A1", "A1")).execute()
```

Comment:
I see what you're doing here, but how does this differ from just providing `A1` once?  Won't they still get concatenated and placed in a single tag?